ckanext/ontario_theme/fanstatic/internal/webassets.yml

```diff
@@ -8,6 +8,7 @@ ontario_theme_common:
     - common.css
     - fonts_googleapis.css
```

Comment:
@Alexandra-Green Do we still need `fonts_googleapis.css` now that we have all the font files in `public/fonts/ds-fonts/`?

Reply:
It doesn't look like we need it anymore, I'll remove it